Changelog.md

```diff
@@ -1,3 +1,49 @@
+### 2.6.0
+(2020-08-10)
+
+
+**新增**
+
+- 新增对Solidity 0.6的支持
+- 新增对国产麒麟操作系统V10（Kunpeng-920/arm aarch64）的支持
+- 新增节点间时间对齐机制
+- 新增对IPv6的支持
+- 新增群组SDK白名单机制，默认关闭
+- 新增获取区块头的接口`getBlockheaderByNumber/getBlockHeaderByHash`
+- 新增支持配置区块执行超时时间
+
+**更改**
+
+- rPBFT共识算法引入vrf随机数算法，进一步保证共识算法的安全性
+- evm引擎由aleth-interpreter替换为evmone
+- 升级EVMC接口版本为7.3.0
+- 源码编译要求GCC版本不小于5.4.0
+- 支持AMOP消息路由到节点自身
+- MySQL存储模式下区块等数据使用二进制存储并启动压缩
+
+**修复**
+
+- 修复因为交易广播标记缓存没有清理干净导致的轻微内存泄露问题
+- 修复raft共识下，非leader节点交易池中的交易不能被打包的问题
+- 在MySQL存储模式下，Solidity合约写接口中调用Table的select接口时，偶现的Gas计算不一致问题
+- 用户自己生成的错误证书导致coredump问题
+- 在Solidity中使用汇编调用以太坊预编译合约0x1-0x9，对错误输入没有返回0的问题
+
+**兼容性**
```

Comment:
Codacy found an issue: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5987732)